src/tasks/SystemsManagerRunCommand/task.json

```
@@ -227,6 +232,24 @@
             "helpMarkDown": "The name of a variable that will contain the unique ID assigned to the command. The command ID can be used future references to the request.",
             "required": false
         },
+        {
+            "name": "cloudWatchOutputEnabled",
+            "label": "Enable CloudWatch Logs",
+            "type": "boolean",
+            "required": false,
+            "defaultValue": false,
+            "groupName": "cloudWatchOutput",
+            "helpMarkDown": "Enables Systems Manager to send command output to CloudWatch Logs."
+        },
+        {
+            "name": "cloudWatchLogGroupName",
+            "label": "CloudWatch Log Group Name",
+            "type": "string",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "cloudWatchOutput",
```

Since this field won't do anything if `cloudWatchOutputEnabled` is false, try adding:

```suggestion
            "groupName": "cloudWatchOutput",
            "visibleRule": "cloudWatchOutputEnabled = true",
```